### PR TITLE
Feature/monitoring service

### PR DESCRIPTION
**Pull request by JeroenStravers1**
Disabled feature toggle, monitoring service is still wip.

### CODE REVIEW

**Comment by etiennedi** [comment]
Hi @JeroenStravers1, the latest build failed on CI because of a data race (see https://travis-ci.org/creativesoftwarefdn/weaviate/builds/498275332#L495-L528). Let me know once this is fixed, then I'm happy to start reviewing :)

**Comment by etiennedi** [comment]
Also, quick info, @JeroenStravers1: Since travis already builds PRs there's no more need to build the branch. (That's why we currently have the travis check twice up there). You can now remove your feature branch from the `.travis.yml` and then we'll have just one build per commit :)

**Comment by JeroenStravers1** [comment]
Ah, I'll remove the Travis build config, good to know!

My latest push should fix the race condition, it existed only in the test case. I missed a line when I refactored the test cases (and conveniently forgot to use the -race flag).

File: telemetry/reporting.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+		r.transformToOutputFormat(extractedLog)
+	}
+}
+
+// Transform the logged function calls to a minimized output format to reduce network traffic
+func (r *Reporter) transformToOutputFormat(logs *map[string]*RequestLog) {
+	transformer := NewOutputTransformer()
+	minimizedLogs := transformer.MinimizeFormat(logs)
+	/*cborLogs := */ transformer.EncodeAsCBOR(minimizedLogs)
+	// TODO:
```
Comment (bobvanluijt) [inline telemetry/reporting.go]: @JeroenStravers1 quick question, will you have issues for todo's?

Comment (JeroenStravers1) [reply telemetry/reporting.go]: [Yes](https://github.com/creativesoftwarefdn/weaviate/issues/736). I figured I could keep my pushes contained to #699, but I noticed that I'm losing some detail/traceability by doing this.

**Pull request closed without merging**